ci-operator/step-registry/ipi/conf/osimage/ipi-conf-osimage-commands.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [[ -z ${PLATFORM} ]]; then
+    echo "The environment variable PLATFORM is not defined!"
+    exit 1
+fi
+
+if [[ -z ${CLUSTER_OS_IMAGE} ]]; then
+    if [[ ! -f "${SHARE_DIR}/cluster_os_image" ]]; then
+        echo "Unable to get CLUSTER_OS_IMAGE!"
+        exit 1
+    fi
+    CLUSTER_OS_IMAGE=$(< "${SHARE_DIR}/cluster_os_image")
+fi
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH="${SHARED_DIR}/install-config-osimage.yaml.patch"
+
+cat > "${PATCH}" << EOF
+platform:
+  ${PLATFORM}:
```

Comment:
How about renaming the step path like `ipi/conf/osimage/azure`, then we can skip to introduce the `PLATFORM` new variable.

Reply:
updated